Tools/scripts/build_binaries.py

```diff
@@ -16,7 +16,7 @@
 import time
 import subprocess
 import sys
-import zlib
+import zlib, gzip
```

Comment:
flake8 will bitch about this.

`build_binaries.py` is currently flake8-clean.  Near-enough:


```
pbarker@bluebottle:~/rc/ardupilot(rally-protocol)$ flake8 Tools/scripts/build_binaries.py 
Tools/scripts/build_binaries.py:161:44: E231 missing whitespace after ','
Tools/scripts/build_binaries.py:162:44: E231 missing whitespace after ','
Tools/scripts/build_binaries.py:164:28: E713 test for membership should be 'not in'
Tools/scripts/build_binaries.py:361:37: E703 statement ends with a semicolon
Tools/scripts/build_binaries.py:415:12: E713 test for membership should be 'not in'
Tools/scripts/build_binaries.py:667:29: E703 statement ends with a semicolon
pbarker@bluebottle:~/rc/ardupilot(rally-protocol)$ 
```

Reply:
fixed